### PR TITLE
[CARBONDATA-3455] Job Group ID is not displayed for the IndexServer Jobs

### PR DESCRIPTION
**Pull request by dhatchayani**
Job Group ID is not displayed for the IndexServer Jobs as it is not set.

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [x] Testing done
        Manual Testing
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3710/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3923/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11986/

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CacheUtil.scala [hunk 1]
```diff
@@ -84,6 +86,22 @@ object CacheUtil {
...
 
+  def getTaskGroupId(sparkSession: SparkSession): String = {
+    val taskGroupId = sparkSession.sparkContext.getLocalProperty("spark.jobGroup.id") match {
+      case null => ""
+      case _ => sparkSession.sparkContext.getLocalProperty("spark.jobGroup.id")
+    }
+    taskGroupId
+  }
+
+  def getTaskGroupDesc(sparkSession: SparkSession): String = {
```
Comment (kunal642) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CacheUtil.scala]: move to SparkSQLUtil

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CacheUtil.scala [hunk 2]
```diff
@@ -84,6 +86,22 @@ object CacheUtil {
     }
   }
 
+  def getTaskGroupId(sparkSession: SparkSession): String = {
```
Comment (kunal642) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CacheUtil.scala]: move to SparkSQLUtil

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3715/

**Comment by kunal642** [comment]
LGTM

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3717/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11993/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3929/

**Comment by kunal642** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3748/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3960/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/12024/

**Comment by brijoobopanna** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3760/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/12039/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3975/

**Comment by kunal642** [comment]
LGTM..CI Failures are because of some jenkins issue

**Pull request closed without merging**